src/components/Activism/index.js

```diff
@@ -0,0 +1,32 @@
+import React, {Component} from 'react';
+import {Row, Col} from 'react-bootstrap';
+import Img from 'react-image';
+import Preloader from '../Preloader/index';
+import Act1 from '../../assets/activism.jpg';
+import GreyBox from '../../assets/greybox.png';
+import './styles.scss';
+
+class Activism extends Component {
+	render () {
+		return (
+			<Row className='actRow'>
+				<Col className='actText'>
+					<p className='actTitle'>
+						ACTIVISM
+					</p>
+					<p className='actDetails'>
+						AAB hopes to inspire a new generation of Asian American leaders, culminating in Reflections, an end-of-year celebration of graduating seniors who have made significant contributions to the Asian American community.
+					</p>
+				</Col>
+				<Col>
```

Comment:
If it's just a singleton like this, then you don't need to put the img in a col element